src/qibolab/platforms/platform.py

```diff
@@ -32,16 +37,41 @@ def disconnect(self):
         self.design.disconnect()
         self.is_connected = False
 
-    def execute_pulse_sequence(self, sequence, nshots=1024, relaxation_time=None, raw_adc=False):
-        if relaxation_time is None:
-            relaxation_time = self.relaxation_time
-        return self.design.play(self.qubits, sequence, nshots=nshots, relaxation_time=relaxation_time, raw_adc=raw_adc)
+    def execute_pulse_sequence(self, sequence, **kwargs):
+        options = ExecutionParameters(**kwargs)
+
+        if options.relaxation_time is None:
+            options.relaxation_time = self.relaxation_time
+        if options.fast_reset:
+            options.fast_reset = {
+                qubit.name: self.create_RX_pulse(qubit=qubit.name, start=sequence.finish)
+                for qubit in self.qubits.values()
+                if not qubit.flux_coupler
+            }
```

Comment:
This will not be needed if we merge #387, but this may take some time so it is okay as a temporary solution. It is just not very good that we are changing the type of `options.fast_reset` from `bool` to `dict`.

Reply:
Also, I don't expect anyone to use this kind of naive qubit reset as it not very goof for the fidilities. So, it can even be removed as it was mainly for being able to benchmark the fast reset.